code/game/objects/items/defibrillator.dm

```diff
@@ -103,6 +103,11 @@
 	user.visible_message(span_notice("[user] turns [src] [ready? "on and opens the cover" : "off and closes the cover"]."),
 	span_notice("You turn [src] [ready? "on and open the cover" : "off and close the cover"]."))
 	playsound(get_turf(src), "sparks", 25, TRUE, 4)
+	if(ready)
+		playsound(get_turf(src), 'sound/items/defib_safetyOn.ogg', 30, 0)
+	else
+		w_class = initial(w_class)
```

Comment:
is this just copy pasted from somewhere? This is not needed.

Reply:
deleting this and the ready sound very very soon